web/track/views.py

```diff
@@ -487,20 +486,22 @@ def verify_account_mobile():
         phone = 'placeholder: ***-***-3333'
         return render_template(generate_path(prefix, "verify-mobile"), phone=phone)
 
-
     def send_pass_reset(user_email, user, prefix):
+        user_name = user.display_name
```

Comment:
Instead of assigning the user.display_name to a variable, can we just put it directly in the personalisation dict? Just make line 500 -> 'user': user.display_name, ?? :)